packages/flutter_tools/test/general.shard/android/gradle_find_bundle_test.dart

```diff
@@ -530,7 +531,7 @@ void main() {
         Event.flutterBuildInfo(
           label: 'gradle-expected-file-not-found',
           buildType: 'gradle',
-          settings: 'androidGradlePluginVersion: 7.6.3, fileExtension: .aab',
+          settings: 'androidGradlePluginVersion: $templateDefaultGradleVersion, fileExtension: .aab',
```

Comment:
Consider using a named import to make it easier to track where this variable comes from. 
`import 'package:flutter_tools/src/android/gradle_utils.dart' as gradleUtils;`

Reply:
Hmm I can change this if you have a strong preference, but it will require many other changes, e.g. every call to `parseGradleVersionFromDistributionUrl`, `getAgpVersion`, `getGradleVersion`, etc.

Reply:
ok maybe leave it alone in this file and add it in the files where you added an import.

Reply:
Done!